src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/Measurement.cs

```diff
@@ -57,23 +57,45 @@ public Measurement(T value, params KeyValuePair<string, object?>[]? tags)
         }
 
         /// <summary>
-        /// Initializes a new instance of the Measurement using the value and the list of tags.
+        /// Initializes a new instance of Measurement with the provided <paramref name="value"/> and a <see cref="ReadOnlySpan{T}"/> containing
+        /// zero or more associated <paramref name="tags"/>.
         /// </summary>
-        /// <param name="value">The measurement value.</param>
-        /// <param name="tags">The measurement associated tags list.</param>
+        /// <param name="value">The value of the measurement.</param>
+        /// <param name="tags">The <see cref="KeyValuePair{TKey, TValue}"/> tags associated with the measurement.</param>
         public Measurement(T value, params ReadOnlySpan<KeyValuePair<string, object?>> tags)
         {
             _tags = tags.ToArray();
             Value = value;
         }
 
         /// <summary>
-        /// Gets the measurement tags list.
+        /// Initializes a new instance of the Measurement with the provided <paramref name="value"/> and a <see cref="TagList"/> containing
+        /// zero or more associated <paramref name="tags"/>.
+        /// </summary>
+        /// <param name="value">The value of the measurement.</param>
+        /// <param name="tags">A <see cref="TagList"/> containing the <see cref="KeyValuePair{TKey, TValue}"/> tags associated with the measurement.</param>
+        public Measurement(T value, in TagList tags)
```

Comment:
Do we have tests for this? I don't see any in this PR.

Reply:
https://github.com/dotnet/runtime/blob/c723f067fddf39528384d65cd79e8a86197555e2/src/libraries/System.Diagnostics.DiagnosticSource/tests/MetricsTests.cs#L25